docs/Options.md

```diff
@@ -195,3 +195,16 @@ Ruby section is shown only in directories that contain `Gemfile`, or `Rakefile`,
 | `SPACEFISH_RUBY_SUFFIX` | `$SPACEFISH_PROMPT_DEFAULT_SUFFIX` | Suffix after Ruby section |
 | `SPACEFISH_RUBY_SYMBOL` | `💎·` | Character to be shown before Ruby version |
 | `SPACEFISH_RUBY_COLOR` | `red` | Color of Ruby section |
+
+### Haskell (`haskell`)
+
+Ruby section is shown only in directories that contain `stack.yaml`.
+
+| Variable | Default | Meaning |
+| :------- | :-----: | ------- |
+| `SPACEFISH_HASKELL_SHOW` | `true` | Show Haskell section |
+| `SPACEFISH_HASKELL_PREFIX` | `$SPACEFISH_PROMPT_DEFAULT_PREFIX` | Prefix before Haskell section |
+| `SPACEFISH_HASKELL_SUFFIX` | `$SPACEFISH_PROMPT_DEFAULT_SUFFIX` | Suffix after Haskell section |
+| `SPACEFISH_HASKELL_SYMBOL` | `"λ "` | Character to be shown before Haskell version |
+| `SPACEFISH_HASKELL_COLOR` | `"red"` | Color of Haskell section |
```

Comment:
Quick note about the default values!
I would remove the quotes within the backticks to keep them consistent with the rest of the docs.
Also, please replace the space after the lambda with a mid-dot: `·`.